src/components/ui/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from 'react';
+
+import styles from './NavBar.module.css';
+import {AppLogo} from "../AppLogo/AppLogo";
+import {Link} from "react-router-dom";
+import {MenuBase} from "../../utils/MenuBase/MenuBase";
+import {NavMenu} from "../NavMenu/NavMenu";
+
+import Burger from '/assets/images/icons/burger.svg';
```

Comment:
[borgir](https://www.youtube.com/watch?v=toZW65rksYY)

Reply:
i will consider changing the name